src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    public MyLinkedList() {
+        this.size = 0;
+    }
```

Comment:
```suggestion
```
you don’t  need to initialize this variable (default  value 0)

Reply:
okay, I get it